src/main/java/org/opensearch/ad/rest/RestGetAnomalyDetectorAction.java

```diff
@@ -105,15 +100,13 @@ protected ExtensionRestResponse prepareRequest(RestRequest request) throws IOExc
         String detectorId = request.param(DETECTOR_ID);
         String typesStr = request.param(TYPE);
 
-        String rawPath = request.path();
-        // FIXME handle this
-        // Passed false until job scheduler is integrated
-        boolean returnJob = false;
-        boolean returnTask = false;
-        boolean all = false;
+        String rawPath = request.rawPath();
```

Comment:
seems this was originally the `request.path()` instead of the `request.rawPath()`. Any particular reason for this change?

Reply:
Because previously we were using `ExtensionRestRequest`. 
`RestRequest` has [request.rawPath()](https://github.com/opensearch-project/anomaly-detection/blob/main/src/main/java/org/opensearch/ad/rest/RestGetAnomalyDetectorAction.java#L64)